### DISCUSSION TITLE
Different feedback cues

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#231] [Help]
I have gotten this working when I made the different dwell times previously, but on this current layout I cannot seem to have two different feedback cues working. 

[magnify_feedback.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/13589850/magnify_feedback.zip)

My goal is that on the "prompt" pages a cursor does appear (in this case the red feedback dot) but on the search pages the magnify feature words. I want this because especially with magnify I do not want the prompt window magnified as they read the sentence. In an ideal world on the "prompt" page there would just be a normal cursor, but was trying first with the feedback cues that I have. 

In my spreadsheet I made state variables for pointer and magnify, but that does not seem to be working, instead somehow I seem to either have one working globally or both simultaneously. This was the same strategy that I used for the different dwell times so not sure why it is not working now. 

For reference these are what I am calling "prompt" pages: 
<img width="250" alt="Screenshot 2023-12-06 at 12 07 33 PM" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/92817124/5c859b24-20f8-4280-b0e5-5cbbdb2ef177">

And these search pages: 
<img width="250" alt="Screenshot 2023-12-06 at 12 07 23 PM" src="https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/92817124/685fc207-b84b-4462-a552-6c69cbbc46f9">

I have also tried using $tabControl = (fill in with tab name) but that doesn't seem to be working for me either. 

I have also seem to have messed up the timing somehow of the dwell??

**Comment by gbishop** [comment]
We should meet online. I'm not confident I know what you want.

**Comment by sofia-bg** [comment]
I wonder if this video helps, on the page with just the sentence and "go" I just want the normal cursor. On the next page with all the symbols I want it to zoom. I almost have it working, but only if the page with go is set to click. Otherwise when I have it with dwell then the zoom function breaks down on the next page. 

[example switch magnify mouse.mp4.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/13640587/example.switch.magnify.mouse.mp4.zip)

[magnify_feedback_12_11_23-3.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/13640593/magnify_feedback_12_11_23-3.zip)

**Comment by gbishop** [comment]
Having two methods active using the same signals is likely to cause
problems. It uses the first-one-wins rule among handlers in the same method
but can't across multiple methods.

I suggest controlling the pointer appearance using Customize. You can
condition the cursor css using (say) the name of the panel or anything else
that makes that "page" unique.

Oh it looks like you're doing that.

**Reply by sofia-bg** [reply]
thanks, i have gotten it close to working on a few of the layouts using customize so good to have feedback that i am on the right track. worst case scenario it's just the same feedback across both types of pages.
